patch/1100-spark_model.json

```diff
@@ -0,0 +1,477 @@
+{
```

Comment:
This is now same as https://github.com/apache/atlas/commit/fce87e338ad3d7a9425f41c09f73968308e8ce83
so unless we fix the model, you can skip reviewing this.

Reply:
Update: this is now same as https://github.com/apache/atlas/blob/2c375b08a52ac8d1039abb4b612a41cb9d89b420/addons/models/1000-Hadoop/1100-spark_model.json